src/main/resources/workflows/0.1.0/irida_workflow.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<iridaWorkflow>
+  <id>4ef5a1ad-435f-4835-b289-deddf0c3f98e</id>
+  <name>STAR_AMR</name>
+  <version>0.1.0</version>
```

Comment:
I'm thinking we should change the version of this workflow to `0.7.0` to mirror that of `staramr`.